core/dbt/version.py

```diff
@@ -44,16 +52,40 @@ def get_version_information():
 
     plugin_version_msg = "Plugins:\n"
     for plugin_name, version in _get_dbt_plugins_info():
-        plugin_version_msg += '  - {plugin_name}: {version}\n'.format(
-            plugin_name=plugin_name, version=version
-        )
+        plugin_version = dbt.semver.VersionSpecifier.from_version_string(version)
+        latest_plugin_version = get_latest_version(version_url=get_package_pypi_url(plugin_name))
+        plugin_update_msg = ''
+        if installed == plugin_version or (
+            latest_plugin_version and plugin_version == latest_plugin_version
+        ):
```

Comment:
It's possible that plugins will put out patch releases _before_ `dbt-core` puts out a patch release. Imagine:
- `dbt-core==1.0.1` and `dbt-synapse==1.0.2` are both available
- I have `dbt-synapse==1.0.1` installed locally
- `installed == plugin_version` will return `True`, at the same time, there's a compatible plugin update available

Reply:
I didn't realize this case, and the rule in the next comment, I will open a new issue and get these changes in